code/modules/hydroponics/grown/dionapod.dm

```diff
@@ -0,0 +1,60 @@
+// A very special plant, deserving it's own file.
+
+/obj/item/seeds/dionapod
+	name = "pack of diona seeds"
+	desc = "These seeds grow into diona nymphs. They say these are used to grow new dionae, just add blood!"
+	icon_state = "seed-dionapod"
+	species = "diona"
+	plantname = "Diona Pod"
+	product = /mob/living/simple_animal/hostile/retaliate/nymph
+	lifespan = 50
+	endurance = 8
+	maturation = 8
+	production = 1
+	yield = 1
+	potency = 30
+	growthstages = 3
+	var/volume = 5
+	var/list/result = list()
+
+/obj/item/seeds/dionapod/harvest(mob/user)
+	var/obj/machinery/hydroponics/parent = src.loc
+	if(CONFIG_GET(flag/diona_ghost_spawn))
```

Comment:
I am not familiar with this define.
Is this cached? If no, cache this, it's easier to not hit config everytime harvest happens.

Reply:
Based off the rest of the code that i've had a look at, it seems to be cached? I am unsure.

Reply:
It's cached